source/slang/slang-ir.h

```diff
@@ -1641,6 +1641,13 @@ struct IRRTTIPointerType : IRRawPointerTypeBase
     IR_LEAF_ISA(RTTIPointerType)
 };
 
+struct IRHLSLConstBufferPointerType : IRPtrTypeBase
+{
+    IR_LEAF_ISA(HLSLConstBufferPointerType)
+    IRInst* getElementType() { return getOperand(0); }
```

Comment:
We already have `getValueType` on IRPtrTypeBase, is the use of element here to in some way distinguish this use case? (Say value might imply a single value, where element implies some amount of?)

Reply:
Ah, you are right, I'll remove it.

Reply:
Fixed.